frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -222,16 +227,17 @@ export default function CreateBooking() {
           </Container>
           <Controller
             name="description"
-            render={({ onChange, value }) => (
+            render={({ onChange, name }) => (
               <InputField
+                name={name}
+                hasError={!!errors.description}
                 title="Description"
                 placeholder="Tell us what your booking is for!"
                 textArea
-                value={value}
                 onChange={onChange}
+                required={true}
```

Comment:
fyi only, can do this
```suggestion
                required
```

Reply:
[966bfbf](https://github.com/rhdevs/rhapp/pull/637/commits/966bfbf1da4390488db7ad28d7d3955be4dc1dad)